src/main/java/rs/teslaris/core/harvester/common/Metadata.java

```diff
@@ -0,0 +1,47 @@
+package rs.teslaris.core.harvester.common;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import rs.teslaris.core.harvester.event.Event;
+import rs.teslaris.core.harvester.organisationunit.OrgUnit;
+import rs.teslaris.core.harvester.patent.Patent;
+import rs.teslaris.core.harvester.person.Person;
+import rs.teslaris.core.harvester.product.Product;
+import rs.teslaris.core.harvester.publication.Publication;
+
+@XmlType(name = "TMetadata")
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "metadata")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class Metadata {
```

Comment:
This class is also not reusable for other formats, should that be renamed to be MetadataOpenAIRECRIS or something as that